frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-node-manager/rpc_checked-commit/readme.md

```diff
@@ -39,235 +39,147 @@ beginning of the transaction.
 
 ### Successful Example
 
-RPC checked-commit input has 2 target nodes and the output describes the
-result of the checked-commit.
+Nodes 'R1' and 'R2' has been changed. RPC checked-commit input invoke all
+touched nodes.
 
 ```bash RPC Request
 curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:checked-commit' \
 --header 'Accept: application/json' \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "input": {
-        "target-nodes": {
-            "node": ["IOSXR","IOSXRN"]
-        }
-    }
-}'
-```
-
-```json RPC Response, Status: 200
-{
-    "output": {
-        "overall-status": "complete",
-        "node-results": {
-            "node-result": [
-                {
-                    "node-id": "IOSXRN",
-                    "configuration-status": "complete"
-                },
-                {
-                    "node-id": "IOSXR",
-                    "configuration-status": "complete"
-                }
-            ]
-        }
-    }
-}
-```
-
-### Successful Example
-
-If the RPC input does not contain the target nodes, all touched nodes
-will be invoked.
-
-```bash RPC Request
-curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:checked-commit' \
---header 'Accept: application/json' \
---header 'Content-Type: application/json' \
---data-raw '{
-    "output": {
-        "overall-status": "complete",
-        "node-results": {
-            "node-result": [
-                {
-                    "node-id": "IOSXRN",
-                    "configuration-status": "complete"
-                },
-                {
-                    "node-id": "IOSXR",
-                    "configuration-status": "complete"
-                }
-            ]
-        }
     }
 }'
 ```
 
 ```json RPC Response, Status: 200
-{
-    "output": {
-        "overall-status": "complete"
-    }
-}
 ```
 
 ### Failed Example
 
-RPC checked-commit input has 2 target nodes. One of them failed on
-configuration (IOSXRN). The output describes the result of the
-checked-commit.
+Nodes 'R1' and 'R2' has been changed. Node 'R1' has failed due to
+improper configuration. The output describes the result of the checked-commit.
```

Comment:
add following info: Both 'R1' and 'R2' are in-sync with actual state on the device.

Reply:
Done.